docs/source/simulated_community.csv

```diff
@@ -0,0 +1,9 @@
+﻿Community,Num. Genomes,Num. Control Sequences

```

Comment:
📝 Add section on using these links. Or simply make some of these commands available to the user
See issue-#110 (assigned @ajlail98). May be worth a discussion b/w @Sidduppal and @ajlail98.

# Download simulated communities 

## Example for the 78Mbp community

```bash
gdown --id 15CB8rmQaHTGy7gWtZedfBJkrwr51bb2y
```
You can get the file ID by navigating to any of the files and right clicking, then selecting the `get link` option. This will have a `copy link` button that you should use.

The link will look like this: `https://drive.google.com/file/d/15CB8rmQaHTGy7gWtZedfBJkrwr51bb2y/view?usp=sharing`

The file ID is within the `/` forward slashes between `file/d/` and `/`, e.g.

```bash
# Pasted from copy link button:
https://drive.google.com/file/d/15CB8rmQaHTGy7gWtZedfBJkrwr51bb2y/view?usp=sharing
#                 begin file ID ^ ------------------------------^ end file ID
```

Now that we have the File ID, you can specify the ID or use the `drive.google.com` prefix

```bash
file_id="15CB8rmQaHTGy7gWtZedfBJkrwr51bb2y"
gdown https://drive.google.com/uc?id=${file_id} -O metagenome.fna.gz
# or
gdown --id 15CB8rmQaHTGy7gWtZedfBJkrwr51bb2y
```

Either should work